components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -224,6 +224,19 @@ class GeckoEngine(
         })
     }
 
+    /**
+     * See [Engine.updateWebExtension].
+     */
+    override fun updateWebExtension(
+        extension: WebExtension,
+        onSuccess: (WebExtension?) -> Unit,
+        onError: (String, Throwable) -> Unit
+    ) {
+        // GeckoView support for updating extensions hasn't been implemented yet
+        // TODO https://github.com/mozilla-mobile/android-components/issues/4500
```

Comment:
nit: I would link to this instead: https://bugzilla.mozilla.org/show_bug.cgi?id=1599581

Reply:
Sure 👍